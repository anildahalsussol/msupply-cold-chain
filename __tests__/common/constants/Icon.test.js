import { ICON } from '~constants';

describe('Constants: ICON', () => {
  it('Matches snapshot', () => {
    expect(ICON).toMatchSnapshot();
  });
});
