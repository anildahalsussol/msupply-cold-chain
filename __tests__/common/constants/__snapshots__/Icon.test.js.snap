// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: ICON Matches snapshot 1`] = `
Object {
  "BATTERY_CHARGING": "battery-charging",
  "BATTERY_EMPTY": "battery-0",
  "BATTERY_FOUR": "battery-4",
  "BATTERY_ONE": "battery-1",
  "BATTERY_THREE": "battery-3",
  "BATTERY_TWO": "battery-2",
  "BLUETOOTH": "bluetooth",
  "CALENDAR": "calendar",
  "CHEVRON_LEFT": "chevron-left",
  "CHEVRON_RIGHT": "chevron-right",
  "CLOSE": "close",
  "COLD_BREACH": "snowflake-o",
  "DOWNLOAD": "download",
  "EMAIL": "envelope-o",
  "HOT_BREACH": "thermometer",
  "LOCK": "lock",
  "LOW_BATTERY": "battery-1",
  "MAP": "map",
  "PENCIL": "pencil",
  "POWER_PLUG_OFF": "power-plug-off",
  "POWER_PLUG_ON": "power-plug",
  "SENSORS": "line-chart",
  "SETTINGS": "cog",
  "SORT": "sort",
  "SORT_DOWN": "sort-desc",
  "SORT_UP": "sort-asc",
  "UNLOCK": "unlock",
}
`;
